docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,137 @@
+In deep learning, training often generates multiple model checkpoints, where the best candidates are further evaluated before production deployment. 
+
+We are introducing a new first-class object, the <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> entity, into MLflow Tracking to streamline these processes. As you define and evaluate your GenAI agents in code, 
+or your deep learning jobs create and evaluate models, they will be automatically stored as MLflow Logged Models in your MLflow Experiment. 
+
+Each GenAI agent consolidates its evaluation metrics from both offline jobs and online interactions if you use managed MLflow with Databricks. 
+This feature removes the need to query Runs for traces from different environments (batch jobs, different notebook sessions etc.) to gain a full understanding of the agent’s performance.
+Similarly, every deep learning checkpoint is stored as a <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> with direct links to its own metrics. There is no longer any need to carefully log evaluation metrics for each checkpoint
+at its epoch all within the same Run, or to later query all metrics at a certain epoch just to retrieve metrics for one checkpoint. 
+
+Specifically, you will be able to:
+- View Logged Models alongside key information, including code and weights as artifacts and configurable parameters that affect model behavior
+- Holistically track all performance metrics related to your logged models with links to training and evaluation jobs
+- Search for and rank existing models based on a set of certain criteria, such as performance metrics on training or evaluation datasets
+
+### Quickstart
+
+This quickstart example demonstrates how to use the LoggedModel lineage feature in MLflow 3.0 with a custom Python model, showcasing its integration with runs and traces.
+
+```python
+import mlflow
+
+
+class MyModel(mlflow.pyfunc.PythonModel):
+    @mlflow.trace
+    def predict(self, model_input: list[str]):
+        return ",".join(model_input)
+
+
+with mlflow.start_run():
+    model_info = mlflow.pyfunc.log_model(
+        python_model=MyModel(), name="my_model", input_example=["a", "b", "c"]
+    )
+
+print(f"LoggedModel model id: {model_info.model_id}")
+
+pyfunc_model = mlflow.pyfunc.load_model(model_info.model_uri)
+pyfunc_model.predict(["x", "y", "z"])
+traces = mlflow.search_traces(model_id=model_info.model_id)
+print(traces)
+
+#                          request_id                                              trace   timestamp_ms  ...                                              spans                                               tags assessments
+# 0  af5da8c78a6f49c793f57ebb19dcb6c7  Trace(request_id=af5da8c78a6f49c793f57ebb19dcb...  1742781867233  ...  [{'name': 'predict', 'context': {'span_id': '0...  {'mlflow.artifactLocation': 'file:///Users/ser...          []
+```
+
+Navigate to the run's page to view the LoggedModel:
```

Comment:
The metrics and params are blank in this example, so the screenshot doesn't look super compelling. Can we include an example that has metrics and params?